extras/nest_vars.sh.in

```diff
@@ -1,26 +1,19 @@
 #!/bin/sh
 
-# NEST is installed here. When you relocate NEST, change this variable.
-export NEST_INSTALL_DIR="@CMAKE_INSTALL_PREFIX@"
+# NEST is installed here. When you relocate NEST, change these variables.
+NEST_LIB_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/nest/"
+NEST_BIN_DIR="@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_BINDIR/"
 
-# NEST finds standard *.sli files $NEST_DATA_DIR/sli
-export NEST_DATA_DIR="${NEST_INSTALL_DIR}/@CMAKE_INSTALL_DATADIR@"
```

Comment:
`NEST_DATA_DIR` aka [`slihomepath`](https://github.com/nest/nest-simulator/blob/master/sli/slistartup.cc#L212) defaults to exactly the same as it is set to here. I thus don't see a problem with removing this superfluous export.